wordpress-mobile/WordPress-Android
# Notifications Refresh (Phase 1)

## Subtasks
### List UI
- https://github.com/wordpress-mobile/WordPress-Android/issues/20022
- https://github.com/wordpress-mobile/WordPress-Android/issues/20023
- https://github.com/wordpress-mobile/WordPress-Android/issues/20021
- https://github.com/wordpress-mobile/WordPress-Android/issues/20020
- https://github.com/wordpress-mobile/WordPress-Android/issues/20039
### Inline Actions
- https://github.com/wordpress-mobile/WordPress-Android/issues/20024
- ~~https://github.com/wordpress-mobile/WordPress-Android/issues/20025~~
- https://github.com/wordpress-mobile/WordPress-Android/issues/20040
### Navigation bar
- https://github.com/wordpress-mobile/WordPress-Android/issues/20066
- https://github.com/wordpress-mobile/WordPress-Android/issues/20065
### Details View
- https://github.com/wordpress-mobile/WordPress-Android/issues/20096
### Push notifications
- https://github.com/wordpress-mobile/WordPress-Android/issues/20099
### Other
- https://github.com/wordpress-mobile/WordPress-Android/issues/20101

-----

## To Test:

Use the test instructions of each PR to test the feature as a whole.
Also check
- Test List UI changes: pdcxQM-39Z-p2
- Navigation Bar and Inline Changes: pdcxQM-3a7-p2

-----

## Regression Notes

1. Potential unintended areas of impact

    - Notifications tab and push notifacations

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

3. What automated tests I added (or what prevented me from doing so)

    - `NotificationsListViewModelTest`, `NotificationHelperTest`

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
